meteor/client/ui/Shelf/Shelf.tsx

```diff
@@ -143,12 +148,20 @@ export class ShelfBase extends React.Component<Translated<IShelfProps>, IState>
 		RundownViewEventBus.on(RundownViewEvents.SWITCH_SHELF_TAB, this.onSwitchShelfTab)
 		RundownViewEventBus.on(RundownViewEvents.SELECT_PIECE, this.onSelectPiece)
 		RundownViewEventBus.on(RundownViewEvents.SHELF_STATE, this.onShelfStateChange)
+
+		if (this.props.fullViewport) {
```

Comment:
I feel like we need to state the reason for why we have this check before we mount the TAKE event (and same for the unmount).
A maintenance developer coming by this in a year have no idea that the reason we have to mount when we are in full screen is because of a "hidden dependency" of RundownView not being loaded

Reply:
Ideally, the check for whether we need to mount the event or not should not be the responsibility of the Shelf (nor the RundownView), but that requires a larger refactor...

Reply:
I have added a comment for it.

Reply:
I have added a comment for it.